.jshintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "maxlen": 250,
+  "globalstrict": true,
+  "predef": [
+    "sinon", 
+    "expect", 
+    "describe", 
+    "angular", 
+    "it",
+    "module", 
+    "beforeEach", 
+    "inject", 
+    "require"
+  ]
+}
```

Comment:
@cxoltero 
Few things:

1. It's preferred to use the `globals` array over `predef` to set these variables since `predef` can *not* be updated inline
2. Take a look at the `environments` section of the JSHint docs. It allows you to include keywords from supported libraries and frameworks. It may not eliminate ALL of these variables, but a good amount should be available with the `environments` array. Take a look here for details: http://jshint.com/docs/options/#environments

What I typically do is add my environments and remove all variables from the `globals` array. Then by running the linter you'll see which variables were *not* included by the environments array.